cypress/e2e/06_phases.cy.js

```diff
@@ -35,7 +35,7 @@ describe('Modify phases name', () => {
                 }).its('body').then(body => {
```

Comment:
@MahtabBukhari change `06_phases.cy.js` to `cypress/e2e/06_phases.cy.ts`, everything else is fine now thanks.